providers/os/id/hypervisor/hypervisor.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package hypervisor
+
+import (
+	"io"
+	"strings"
+
+	"github.com/rs/zerolog/log"
+	"go.mondoo.com/cnquery/v11/providers-sdk/v1/inventory"
+	"go.mondoo.com/cnquery/v11/providers/os/connection/shared"
+	"go.mondoo.com/cnquery/v11/providers/os/resources/powershell"
+)
+
+// list of known hypervisors
+var knownHypervisors = map[string]string{
+	"vmware":     "VMware",
+	"virtualbox": "VirtualBox",
+	"kvm":        "KVM",
+	"qemu":       "QEMU",
+	"hyper-v":    "Hyper-V",
+	"xen":        "Xen",
+	"parallels":  "Parallels",
+	"bhyve":      "bhyve",
+	"proxmox":    "Proxmox VE",
+	"openvz":     "OpenVZ",
+	"virtuozzo":  "Virtuozzo",
+	"powervm":    "IBM PowerVM",
+	"lxc":        "LXC/LXD",
```

Comment:
I think we might want to skip this one since these aren't really VMs. That was a mistake from a past life there